src/app/zap-templates/templates/app/ids/Commands.zapt

```diff
@@ -20,6 +20,7 @@ namespace Commands {
 {{/first}}
 namespace {{asUpperCamelCase label}} {
 static constexpr CommandId Id = {{asMEI manufacturerCode code}};
+static constexpr bool kRequiresLargePayloadTransport = {{isLargeMessage}};
```

Comment:
This does not seem like the right place for this.  This sort of metadata generally lives in cluster-objects, not the ids header.  The one really is just for the ids.

Reply:
This looks like something that should eventually be plumbed through https://github.com/project-chip/connectedhomeip/blob/master/src/app/data-model-provider/MetadataTypes.h#L86 or similar (not sure if this is a quality or not, but it sure looks like a boolean flag on commands)

Reply:
This is how the codegen data model currently loads those: https://github.com/project-chip/connectedhomeip/blob/master/src/app/codegen-data-model-provider/CodegenDataModelProvider.cpp#L233

It seems like you may want to plumb this similarely to `CommandNeedsTimedInvoke` and `CommandIsFabricScoped`.

Reply:
Yes, looking at the auto-gen'ed code, it felt this was probably not the right place. Was looking to find the appropriate location. Thanks for the pointers.

Reply:
Modified to a `CommandIsLarge` method in the cluster-objects zapt

Reply:
So the key here is that we will need to go from a dynamic command id to whether it's large, not from a statically known command name to whether it's large, right?  @pidarped

Reply:
Yes, I think so. But to also note, that ultimately the `L` quality is gleaned from what is specified in the xml for the Command.